en/docs/learn/consent-management-try-out.md

```
@@ -149,31 +149,29 @@ The bank sends the authorization request to the customer in the following URL fo
+6. Upon providing consent, an authorization code is generated in the `redirect_uri` URL.
 
 **Sample Authorization Code Response:**
 
 The authorization code is in the `code` parameter (`code=e61579c3-fe9c-3dfe-9af2-0d2f03b95775`):
   
-      ```
-      https://wso2.com/#code=e61579c3-fe9c-3dfe-9af2-0d2f03b95775&id_token=eyJ4NXQiOiJNell4TW1Ga09HWXdNV0kwWldObU5EY3hOR1l3WW1
-      NNFpUQTNNV0kyTkRBelpHUXpOR00wWkdSbE5qSmtPREZrWkRSaU9URmtNV0ZoTXpVMlpHVmxOZyIsImtpZCI6Ik16WXhNbUZrT0dZd01XSTBaV05tTkRjeE5
-      a4f936c74e2ca7f4250208aa42.sk_04ejciXBj6DnpALyYaw
-      ```
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)